js/src/lib/components/InferenceWidget/shared/WidgetWrapper/WidgetWrapper.svelte

```diff
@@ -1,6 +1,6 @@
-<script lang="ts">
+<script lang="ts" generics="T extends WidgetExample">
```

Comment:
just wondering why a generic vs just `WidgetExample` here, since WidgetExample is already one of the defined type?

Reply:
i think the compiler complains if i do that (in the importing files)